### PR TITLE
Sanity check video id

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
Before making request to brightcove

https://app.getsentry.com/nextftcom/ft-next-frontend/group/56963178/

### CODE REVIEW

**Comment by commuterjoy** [comment]
it's worth reporting this as an upstream problem isn't it? where are the bad video id's coming from?

**Comment by ironsidevsquincy** [comment]
Reported and fixed. I think this is still a useful sanity check to have

**Comment by matthew-andrews** [comment]
my preference would be not to fix it in grumman if it's already fixed upstream

**Comment by matthew-andrews** [comment]
oh wait this just throws an error, that's fine.  will make it easier to debug if reoccurs…

File: client/components/video/brightcove.js [hunk 1]
```diff
@@ -2,6 +2,11 @@
 'use strict';
 
 function brightcove(videoId) {
+
+	if (!videoId.match(/^\d{13}$/)) {
+		return Promise.reject(new Error('Invalid video id: ' + videoId));
```
Comment (matthew-andrews) [inline client/components/video/brightcove.js]: any reason why you do this an not
```js
throw new Error('Invalid video id: ' + videoId);
```

Comment (ironsidevsquincy) [inline client/components/video/brightcove.js]: pretty sure it didn't work for some reason, got the `Uncaught (in promise)...` error in the console

Comment (matthew-andrews) [inline client/components/video/brightcove.js]: weird, that should work.  would you mind showing me?

Comment (ironsidevsquincy) [inline client/components/video/brightcove.js]: Rather, https://github.com/Financial-Times/next-grumman/blob/master/client/components/video/main.js#L27 fails when calling `then` on a returned `Error` object

Comment (matthew-andrews) [inline client/components/video/brightcove.js]: ah, it shouldn't be `return new Error`, should be just `throw new Error(…);`

**Pull request merged**